RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -223,15 +250,35 @@ boolean L12_preOutfit()
 		}
 	}
 
-	if(my_level() < 12)
+	// if we want to do war on the fratboy side, then we need to adventure in hippy camp for [filthy hippy disguise] outfit to then adventure in frat house for frat war outfit
+	if(!get_property("auto_hippyInstead").to_boolean())
 	{
-		autoAdv(1, $location[Hippy Camp]);
+		auto_log_info("Trying to acquire a filthy hippy outfit", "blue");
+		if(my_level() < 12)

```

Comment:
This is not a good idea. Check if the level 12 quest has been started or not which is what changes the Island locations. Just being level 12 is not what triggers that (you have to go speak to the council first to start the quest).
`internalQuestStatus("questL12War") > -1` is a better condition.
This may seem like it's splitting hairs but when a path like KoE comes along and we're still waiting to reach specific levels before we do things it will require us to sift through the code and fix stuff like this (see PR #207 for when I did this for KoE, it was not fun).

Reply:
change made